packages/account/src/Sections/Security/DeactivateAccount/account-has-balance.jsx

```diff
@@ -21,8 +21,12 @@ const Content = ({ currency_icon, loginid, title, value }) => (
         <div className='deactivate-account-error__account-details'>
             <Icon icon={currency_icon} size={24} />
             <div className='deactivate-account-error__account'>
-                <span className='deactivate-account-error__accounts-currency'>{title}</span>
-                <span className='deactivate-account-error__accounts-loginid'>{loginid}</span>
+                <Text className='deactivate-account-error__accounts-currency' color='prominent' size='xs'>
```

Comment:
We can remove the class:
`deactivate-account-error__accounts-currency` from the text component and also from sass file.

Reply:
Could you please check it for the rest of changes? I mean the important point for replacing is removing extra styles from sass files.

Reply:
Yes I will